### ISSUE TITLE
Display Custom Locktime On Trezor Screen for BTC

### ISSUE THREAD

**Issue by yuriygeorge** [trezor/trezor-core#506]
I'm a Trezor app developer and for my application, I need to be able to set the locktime parameter, i.e., so the transaction is signed by Trezor and is only valid when a specific block number is reached. However, I notice that the custom "locktime" field is not displayed on the Trezor display screen when set. I understand that when it is not set, no need to display it. But for my application when I need to set the locktime, how can the user be certain that the locktime field was actually set? Is it possible in the near future to make Trezor display the locktime field if it is set? Otherwise, seems to defeat the purpose if the user isn't able to verify it on the trezor device.

**Comment by tsusanka** [comment]
TLDR: if locktime is different than the default value add a screen for confirming it

**Comment by ciny** [comment]
Based on discussion in meeting: we'll be displaying raw value and differentiate between blockheight/timestamp

locktime:
blockheight: 571032

or

locktime:
timestamp: 500 000 001

**Comment by tsusanka** [comment]
So, @prusnak should we consider 0xFFFFFFFF and 0xFFFFFFFE as allowed values without confirmation? I believe so

**Comment by prusnak** [comment]
@tsusanka I got that mixed. 0xFFFFFFFF and 0xFFFFFFFE are nSequence values of the input. Transaction is RBF when all inputs have nSequence == 0xFFFFFFFE. No work is need to be done.

**Comment by yuriygeorge** [comment]
Thanks!
